### DISCUSSION TITLE
Notify webhooks that they have been deleted

### DISCUSSION THREAD

**Discussion by HerrLevin** [Traewelling/traewelling#2006] [Ideas / Feature Requests]
As a webhook developer I want to be notified that my access has been revoked.

This could be implemented via an additional webhook event, or a specific payload in the regular webhook.

              Maybe we should notify the webhook endpoint with an event that the webhook has been deleted?

_Originally posted by @NyCodeGHG in https://github.com/Traewelling/traewelling/issues/1453#issuecomment-1475209722_

**Comment by jheubuch** [comment]
Additionally as a webhook service I want to be able to self-check whether the webhook was deleted on Träwelling-side or if it is still active.
